client/verta/tests/unit_tests/test_deployed_model_class.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+""" Unit tests for the verta.deployment.DeployedModel class. """
+
+import os
+import pytest
+import responses
```

Comment:
Is `responses` new? It'll need to be added to [the unit test requirements](https://github.com/VertaAI/modeldb/blob/main/client/verta/requirements-unit-tests.txt).